### ISSUE TITLE
QML example with GUI

### ISSUE THREAD

**Issue by bomm** [GENIVI/CANdevStudio#246]
Could you, please, provide a QMLExecutor example script with a simple GUI, for example an extended version of `simple_forwarder.qml` with an on-off switch to enable or disable the forwarding or to change the `frameId` or `signalValue` offset?

**Comment by phatpaul** [comment]
Yes please!  I just want to make a quick GUI to test a CAN device that I'm developing on a microcontroller.  
This looks very promising, but I have no experience with QML.
A sample GUI with a few buttons and indicators would save me a lot of learning about QML and GUI development.
